docs/site/Context.md

```diff
@@ -293,7 +332,7 @@ come and go. There are a few caveats associated with that:
      .to('foo-value')
      .tag('foo-tag');
    ctx.add(binding);
-   ctx.on('bind', binding => {
+   ctx.on('bind', {binding} => {
```

Comment:
Hmm, I didn't realize we already have context-event-listener contract with positional arguments. I think the switch from positional arguments to ContextEvent object can break existing code. Do we want to release this as a semver-major version?

Reply:
Marked it as breaking change.